llvm-runtimes/compiler-rt/compiler-rt-22.0.0.9999.ebuild

```diff
@@ -82,6 +85,18 @@ src_configure() {
 		fi
 
 		strip-unsupported-flags
+
+		# The full clang configuration might not be ready yet. Given that compiler-rt
+		# require runtime, use only the linker configuration.
+		local flags=(
+			--no-default-config
+			-nodefaultlibs
+			-lc
```

Comment:
These are already added below. I don't exactly mind having them added unconditionally, but in that case the redundant bits below should be removed.

Reply:
After reading through https://bugs.gentoo.org/862540#c16, I removed them from here and kept the logic below.